src/transformers/models/layoutlm/modeling_layoutlm.py

```diff
@@ -638,8 +640,10 @@ def _init_weights(self, module):
 
             `What are input IDs? <../glossary.html#input-ids>`__
         bbox (:obj:`torch.LongTensor` of shape :obj:`({0}, 4)`, `optional`):
-            Bounding Boxes of each input sequence tokens. Selected in the range ``[0,
-            config.max_2d_position_embeddings-1]``.
+            Bounding boxes of each input sequence tokens. Selected in the range ``[0,
+            config.max_2d_position_embeddings-1]``. Each bounding box should be a normalized version in (x0, y0, x1,
+            y1) format, where (x0, y0) corresponds to the position of the upper left in the bounding box, and (x1, y1)
+            represents the position of the lower right. See Overview section for normalization.
```

Comment:
Could we link the "Overview section for normalization" here directly? Or maybe put this in the glossary. It would be very nice if it's absolutely trivial for the user to know how to preprocess his image to get the bbox

Reply:
Linked it to the Overview section